lightningd/subd.h

```diff
@@ -96,24 +96,24 @@ void subd_send_fd(struct subd *sd, int fd);
  * @sd: subdaemon to request
  * @msg_out: request message (can be take)
  * @fd_out: if >=0 fd to pass at the end of the message (closed after)
- * @fd_in: if not NULL, where to put fd read in at end of reply.
+ * @num_fds_in: how many fds to read in to hand to @replycb.
  * @replycb: callback when reply comes in, returns false to shutdown daemon.
  * @replycb_data: final arg to hand to @replycb
  *
  * @replycb cannot free @sd, so it returns false to remove it.
  */
-#define subd_req(sd, msg_out, fd_out, fd_in, replycb, replycb_data)	\
```

Comment:
This is a great change, I was having trouble passing back the `owner` socket fd from the gossip to the main daemon upon release since it only supported a single fd, I might go and change that now ^^